optimum/exporters/onnx/__main__.py

```diff
@@ -194,10 +202,21 @@ def main():
                 atol=args.atol,
             )
 
-    except ValueError:
-        logger.error(f"An error occured, but the model was saved at: {args.output.parent.as_posix()}")
-        return
-    logger.info(f"All good, model saved at: {args.output.parent.as_posix()}")
+        logger.info(f"The ONNX export succeeded and the exported model was saved at: {args.output.parent.as_posix()}")
+    except ShapeError as e:
+        raise ShapeError(e)
```

Comment:
Quick question, not sure about it:
```suggestion
        raise e
```
?

Reply:
Thank you!